### PR TITLE
Packages chartjs.0.2.2, chartjs-annotation.0.2.2, chartjs-colorschemes.0.2.2, chartjs-datalabels.0.2.2 and chartjs-streaming.0.2.2

### PR DESCRIPTION
**Pull request by monstasat**
This pull-request concerns:
-`chartjs.0.2.1`: OCaml bindings for Chart.js
-`chartjs-annotation.0.2.1`: OCaml bindigns for Chart.js annotation plugin
-`chartjs-colorschemes.0.2.1`: OCaml bindigns for Chart.js colorschemes plugin
-`chartjs-datalabels.0.2.1`: OCaml bindigns for Chart.js datalabels plugin
-`chartjs-streaming.0.2.1`: OCaml bindings for Chart.js streaming plugin

---
* Homepage: https://github.com/monstasat/chartjs-ocaml
* Source repo: git+https://github.com/monstasat/chartjs-ocaml.git
* Bug tracker: https://github.com/monstasat/chartjs-ocaml/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
You didn't use `opam-publish` correctly. Please refer to the README: https://github.com/ocaml/opam-publish/blob/master/README.md (using `dune-release` instead of `opam-release` might help too)

File: packages/chartjs-annotation/chartjs-annotation.0.2.1/opam [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/chartjs-annotation/chartjs-annotation.0.2.1/opam]:
```suggestion
  "dune" {>= "1.11"}
```
Dune should not be tagged as a build dependency (see #14266). Furthermore the `dune-project` in your repository says that this package requires at least dune 1.11

File: packages/chartjs-colorschemes/chartjs-colorschemes.0.2.1/opam [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/chartjs-colorschemes/chartjs-colorschemes.0.2.1/opam]:
```suggestion
  "dune" {>= "1.11"}
```

File: packages/chartjs-datalabels/chartjs-datalabels.0.2.1/opam [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/chartjs-datalabels/chartjs-datalabels.0.2.1/opam]:
```suggestion
  "dune" {>= "1.11"}
```

File: packages/chartjs-streaming/chartjs-streaming.0.2.1/opam [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/chartjs-streaming/chartjs-streaming.0.2.1/opam]:
```suggestion
  "dune" {>= "1.11"}
```

File: packages/chartjs/chartjs.0.2.1/opam [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "dune" {build}
```
Comment (kit-ty-kate) [inline packages/chartjs/chartjs.0.2.1/opam]:
```suggestion
  "dune" {>= "1.11"}
```

File: packages/chartjs-annotation/chartjs-annotation.0.2.1/opam [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  "js_of_ocaml"
+  "js_of_ocaml-ppx"
+  "chartjs"
```
Comment (kit-ty-kate) [inline packages/chartjs-annotation/chartjs-annotation.0.2.1/opam]:
```suggestion
  "chartjs" {= version}
```
Are you sure there is no minimum and maxium version for this dependency? I'd recommend to use `= version` to match every releases together.

File: packages/chartjs-colorschemes/chartjs-colorschemes.0.2.1/opam [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+  "js_of_ocaml"
+  "js_of_ocaml-ppx"
+  "chartjs"
```
Comment (kit-ty-kate) [inline packages/chartjs-colorschemes/chartjs-colorschemes.0.2.1/opam]:
```suggestion
  "chartjs" {= version}
```

File: packages/chartjs-datalabels/chartjs-datalabels.0.2.1/opam [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  "js_of_ocaml"
+  "js_of_ocaml-ppx"
+  "chartjs"
```
Comment (kit-ty-kate) [inline packages/chartjs-datalabels/chartjs-datalabels.0.2.1/opam]:
```suggestion
  "chartjs" {= version}
```

File: packages/chartjs-streaming/chartjs-streaming.0.2.1/opam [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+  "js_of_ocaml"
+  "js_of_ocaml-ppx"
+  "chartjs"
```
Comment (kit-ty-kate) [inline packages/chartjs-streaming/chartjs-streaming.0.2.1/opam]:
```suggestion
  "chartjs" {= version}
```

**Comment by monstasat** [comment]
Sorry for the mistakes, it's my first experience adding a project to opam repository. I'll fix the mentioned issues soon.

**Comment by avsm** [comment]
no apologies needed -- take as many goes as you need over here until you feel it's ready. That's why the CI system exists :-)

**Comment by monstasat** [comment]
A new PR will be created for the new package version

**Pull request closed without merging**